app/models/Prepaid.coffee

```diff
@@ -83,13 +83,29 @@ module.exports = class Prepaid extends CocoModel
       # no includedCourseIDs means full-license, so always return true
       return courseID in (@get('includedCourseIDs') ? [ courseID ])
 
+  numericalCourses: ->
+    return 2047 unless @get('includedCourseIDs')?.length
```

Comment:
what does 2047 represent? It's better to have constants than magic-numbers

Reply:
oh good point